tests/chains/ethereum/contract/truffle-config.js

```diff
@@ -0,0 +1,45 @@
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const mnemonic =
+  "math razor capable expose worth grape metal sunset metal sudden usage scheme";
+
+module.exports = {
+  networks: {
+    chain0: {
+      network_id: "*",
+      provider: () =>
+        new HDWalletProvider({
+          mnemonic: {
+            phrase: mnemonic,
+          },
+          providerOrUrl: "http://127.0.0.1:8645",
+          addressIndex: 0,
+          numberOfAddresses: 10,
+        }),
+    },
+    chain1: {
+      network_id: "*",
+      provider: () =>
+        new HDWalletProvider({
+          mnemonic: {
+            phrase: mnemonic,
+          },
+          providerOrUrl: "http://127.0.0.1:8745",
+          addressIndex: 0,
+          numberOfAddresses: 10,
+        }),
+    },
+  },
+
+  compilers: {
+    solc: {
+      version: "0.6.8",
```

Comment:
Shouldn't you specify a newer version of solc?

Reply:
Change to 0.8.9
https://github.com/datachainlab/yui-relayer-build/commit/9ad5aca503830fc1d084dc839397f39de10ad8bb